R/run_pacta.R

```diff
@@ -0,0 +1,57 @@
+#' Run PACTA for all portfolios matching a pattern in an input directory
+#'
+#' @param path String. Directory where portfolios are located.
+#' @param regext String. Regular expression that defines a valid portfolio name.
+#'
+#' @examples
+#' run_pacta()
+#' @noRd
+run_pacta <- function(path = "../input", regexp = "_Input[.]csv") {
+  portfolios <- portfolios(path, regexp)
+  pacta_source <- write_pacta_source()
+  command <- glue::glue("Rscript --vanilla {pacta_source} {portfolios}")
+
+  for (i in seq_along(command)) {
+    message("Start portfolio: ", portfolios[[i]])
+    system(command[[i]])
+    message("End portfolio: ", portfolios[[i]])
+  }
+}
```

Comment:
Comment from @maurolepore 
> One simple way to avoid the verbose output is by calling system() in an external R process with:
> 
> ``` r
>  callr::r(system, list(command[[i]]))
> ```
> The problem is that it goes too far to the other end and gives too little feedback for a too-long process. Maybe we can add a progress bar with the progress package?

Reply:
I don't know why but I no longer see the annoying output I was trying to hide.